### PR TITLE
Remove deployStrategy from samples and docs

### PR DESCRIPTION
**Pull request by rabi**
Depends-On: https://github.com/openstack-k8s-operators/install_yamls/pull/546

### CODE REVIEW

**Review by fao89 — APPROVED** [review]

**Review by slagle — COMMENTED** [review]
+2.

This probably shouldn't have the lgtm label until the install_yamls actually merges. Otherwise, this one could merge first which would break local install_yamls environment since the samples are taken from a git checkout of dataplane-operator.

**Comment by rabi** [comment]
recheck

depends-on merged.

**Pull request merged**
